docs/conf.py

```diff
@@ -218,7 +224,12 @@
 
 # -- Stability Page ------------------------------------------------------------
 
-with open('./dev_guide/sunpy_stability.yaml', 'r') as estability:
+if is_development == True:
+    url = './dev_guide/sunpy_stability.yaml'
+else:
+    url = './dev_guide_release/sunpy_stability.yaml'
```

Comment:
I am not sure this is what we are after. 

The goal is prevent the removal of the dev_guide when we do a release. 

The stability page should be moved wholesale.

Reply:
The goal was to automatically remove all the files inside dev_guide (except index.rst)  whenever a stable version is released. So I made 2 folders one for stability release and one for dev release and this url tells the location of the `stability.yaml` file. Is this approach wrong?

Reply:
Does this work? 

I think we want to move the sunpy_stability page regardless of if its a release or not.